docs/frontend_utilisation_guide.md

```diff
@@ -24,48 +24,91 @@ The Pixel Streaming Frontend UI contains all the functionality for UI components
 
 ---
 
-## Getting Started
-### Installing the Scalable Pixel Streaming Frontend   
+## Scalable Pixel Streaming Frontend Pieces
+### The Library
```

Comment:
Thoughts on changing this title to one of:

- Scalable Pixel Streaming Frontend library
- The library

Reply:
resolved in bb347ef46c9ddfde8aa1719ffede43fdaf998764